openexr.yaml

```diff
@@ -0,0 +1,107 @@
+# Generated from https://git.alpinelinux.org/aports/plain/community/openexr/APKBUILD
+package:
+  name: openexr
+  version: 3.2.4
+  epoch: 0
+  description: High dynamic-range image file format library
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - boost-dev
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - chrpath
+      - cmake
+      - imath-dev
+      - libdeflate-dev
+      - libdeflate-static
+      - linux-headers
+      - samurai
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/openexr/openexr
+      expected-commit: a1a00ffeecf627bbbc010d40700720bee48b2af7
+      tag: v${{package.version}}
+
+  - uses: cmake/configure
+    with:
+      opts: |
+        -DBUILD_TESTING=OFF \
+        -DBUILD_SHARED_LIBS=True
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - name: openexr-doc
+    pipeline:
+      - uses: split/manpages
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/share
+          mv ${{targets.destdir}}/usr/share/doc ${{targets.subpkgdir}}/usr/share
+    description: openexr manpages
+
+  - name: openexr-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - openexr
+    description: openexr dev
+
+  - name: openexr-libiex
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/lib
+          mv "${{targets.destdir}}"/usr/lib/libIex-*.so.* "${{targets.subpkgdir}}"/usr/lib
+    description: openexr libiex
+
+  - name: openexr-libilmthread
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/lib
+          mv "${{targets.destdir}}"/usr/lib/libIlmThread-*.so.* "${{targets.subpkgdir}}"/usr/lib
+    description: openexr libilmthread
+
+  - name: openexr-libopenexr
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/lib
+          mv "${{targets.destdir}}"/usr/lib/libOpenEXR-*.so.* "${{targets.subpkgdir}}"/usr/lib
+    description: openexr libopenexr
+
+  - name: openexr-libopenexrcore
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/lib
+          mv "${{targets.destdir}}"/usr/lib/libOpenEXRCore-*.so.* "${{targets.subpkgdir}}"/usr/lib
+    description: openexr libopenexrcore
+
+  - name: openexr-libopenexrutil
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/lib
+          mv "${{targets.destdir}}"/usr/lib/libOpenEXRUtil-*.so.* "${{targets.subpkgdir}}"/usr/lib
+    description: openexr libopenexrutil
+
+update:
```

Comment:
@mesaglio can we add some basic tests 

there are lot of cmd: provides which can improve the confidence of package builds and updates 

``` cmd:exr2aces=3.2.4-r0
2024/05/13 17:43:42 INFO     cmd:exrenvmap=3.2.4-r0
2024/05/13 17:43:42 INFO     cmd:exrheader=3.2.4-r0
2024/05/13 17:43:42 INFO     cmd:exrinfo=3.2.4-r0
2024/05/13 17:43:42 INFO     cmd:exrmakepreview=3.2.4-r0
2024/05/13 17:43:42 INFO     cmd:exrmaketiled=3.2.4-r0
2024/05/13 17:43:42 INFO     cmd:exrmanifest=3.2.4-r0
2024/05/13 17:43:42 INFO     cmd:exrmultipart=3.2.4-r0
2024/05/13 17:43:42 INFO     cmd:exrmultiview=3.2.4-r0
2024/05/13 17:43:42 INFO     cmd:exrstdattr=3.2.4-r0
```

Reply:
Adding on https://github.com/wolfi-dev/os/pull/19178/commits/a9dd6b8d6d1cbab2e6378fa5bdcab4960a31831d

Reply:
Thanks @mesaglio